AllReadyApp/Web-App/AllReady/Views/Shared/_LoginPartial.cshtml

```diff
@@ -4,10 +4,10 @@
 
 @if (User.Identity.IsAuthenticated)
 {
-    <form asp-controller="Account" asp-action="LogOff" method="post" id="logoutForm">
+    <form asp-controller="Account" asp-action="LogOff" method="post" id="logoutForm"> 
         <ul class="nav navbar-nav navbar-right login-status">
             <li>
-                <a asp-controller="Manage" asp-action="Index" title="Manage">Hello @User.GetDisplayName()!</a>
+                <a asp-controller="Manage" asp-action="Index" title="Manage">Hello @User.GetDisplayName() @if(User.IsUserProfileIncomplete() ) { <i class="fa fa-exclamation-circle text-info user-profile-warning" title="Click here to add more details to your profile"></i>} </a>                

```

Comment:
The contrast between the blue-ish warning icon and the orange background is not great. I'd suggest removing the `text-info` class. If you want the icon to be more subdued, I'd suggest using a light gray color or adding some opacity. 0.7 seems to work nicely.
![image](https://cloud.githubusercontent.com/assets/775229/13198010/069899f4-d7c3-11e5-8a7d-3f68f43115a8.png)